app/src/main/java/org/inspire/breath/activities/Questions.java

```diff
@@ -0,0 +1,26 @@
+package org.inspire.breath.activities;
+
+public class Questions {
+
+    public String mQuestions [] = {
+            "Do you have more then 3 loose stools /24h?",
+            "3 weeks without blood or rice water aspect?",
+            "Signs of severe dehydration: Sunken eyes, Skin pinch >3 seconds or other danger signs?"
+    };
+
+    public String mAnswers [] = {
```

Comment:
Since this class will only be used once to describe the questions it makes more sense for these arrays to be member values of the Activity